### ISSUE TITLE
Remove Jellyfin Password Reset

### ISSUE THREAD

**Issue by cremonies** [Fallenbagel/jellyseerr#212]
### Description

When Reset Password is clicked, Jellyseerr redirects user to LAN IP and Port for Jellyfin.

### Desired Behavior

Remove Option. User should reset password from Jellyfin UI.

### Additional Context

Docker
Bridged Network
Reverse Proxy

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

**Comment by Fallenbagel** [comment]
The forget password will redirect to the external hostname if set though

**Comment by cremonies** [comment]
Jellyseerr>settings>jellyfin>external URL?
I have it pointing to my subdomain.

**Comment by Fallenbagel** [comment]
> Jellyseerr>settings>jellyfin>external URL?
> I have it pointing to my subdomain.

Is this on stable release? I think I added the logic recently and did not release to stable yet

**Comment by cremonies** [comment]
Stable release. 
I will

**Comment by Fallenbagel** [comment]
#199 It seems that this is a duplicate issue.

I will push the fix tonight

**Comment by adyanth** [comment]
Is this fixed? I still see https://github.com/Fallenbagel/jellyseerr/blob/9252817b5823d70c551843b51158cdd3d57cd415/src/components/Login/JellyfinLogin.tsx#L284

**Comment by Fallenbagel** [comment]
> Is this fixed? I still see

Not yet

**Comment by jakanaka** [comment]
I was going to post an issue on this, but seems it already exists. facing the same problem with latest docker release

**Comment by Fallenbagel** [comment]
> I was going to post an issue on this, but seems it already exists. facing the same problem with latest docker release

Yeah the button will be removed. Password resets should be done on jellyfin/jfa-go for jellyfin accounts

**Comment by pichouk** [comment]
Gentle up on this issue :sweat_smile: 
To be honest I tried to take a look by myself to fix and use the external URL on the Forgot Password button, but I'm 100% newbie on this project and I didn't figure it out. But maybe I can try to submit a PR for removing the button if it can help ?

**Comment by Fallenbagel** [comment]
> Gentle up on this issue :sweat_smile: 
> To be honest I tried to take a look by myself to fix and use the external URL on the Forgot Password button, but I'm 100% newbie on this project and I didn't figure it out. 

I actually forgot I was gon remove the button 😅

> But maybe I can try to submit a PR for removing the button if it can help ?

Feel free to PR for a removal c:

**Comment by Hani-K** [comment]
The issue still persists, however it would be better to allow using a custom link for the button rather than hard linking it to [link]/web/index.html#!/forgotpassword.html. I'm using an LDAP server (Authentik) which handles user authentication and I direct my Jellyfin users to it to when they need to change/recover passwords.

**Comment by Fallenbagel** [comment]
> The issue still persists, however it would be better to allow using a custom link for the button rather than hard linking it to [link]/web/index.html#!/forgotpassword.html. I'm using an LDAP server (Authentik) which handles user authentication and I direct my Jellyfin users to it to when they need to change/recover passwords.

Current plans are to just remove the button.

**Comment by noybman** [comment]
I'm running 1.8.1 (docker image) and the button is still there on the login page. I use Emby, when clicked, it takes you to the Emby password reset.
If the button should be removable, I do not see a way to remove it.
I chose to put the external url to https://127.0.0.1    :)

I think this reset button is a nice feature, but its only useful if the app/link it is redirecting you to is usefully working. More importantly, password resets can be a secuirty risk and thus, some people don't want this feature. (I had tp put a valid url in .... so I did).
